README.md

```diff
@@ -48,6 +48,11 @@ curl -L https://github.com/DataBiosphere/terra-cli/releases/latest/download/down
 ./terra
 ```
 
+By default, the CLI will be installed without support for Docker (i.e. it won't pull the Docker image).
```

Comment:
Also update the example usage section, since those commands will no longer work with a default installation unless the user has setup their environment exactly.